frontend/packages/client/src/components/HomeHeader.js

```diff
@@ -1,17 +1,44 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { useStarAnimation } from 'hooks';
+
+const stars = [
+  { topPer: 0, leftPer: 0, width: '31px', height: '31px', start: 100 },
+  { topPer: 0.42, leftPer: 0.18, width: '20px', height: '20px', start: 1000 },
+  { topPer: 0.1, leftPer: 0.45, width: '31px', height: '31px', start: 700 },
+  { topPer: 0.1, leftPer: 0.99, width: '35px', height: '35px', start: 1100 },
+  { topPer: 0.35, leftPer: 0.7, width: '31px', height: '31px', start: 1100 },
+  { topPer: 0.9, leftPer: 0.7, width: '25px', height: '25px', start: 500 },
+];
 
 export default function HomeHeader() {
+  const { addToArrayOfRefes, starArray, parentRef } = useStarAnimation({
+    stars,
+  });
```

Comment:
probably makes more sense to initialize `parentRef` in `HomeHeader` and then pass it to `useStarAnimation` as an argument. then `useStarAnimations` would just return `starArray` which you can render in the JSX wherever you want